system-test/env.example

```diff
@@ -0,0 +1,16 @@
+# Example env file that can be used to override env vars for running system-test. Copy this file and
+# rename to `env` for the Makefile to use it.
+#
+# Any variable not supplied here will default to the Makefile's values or can be supplied via
+# enviornment variable. You do not need to define everything here.
+#
+# Using ?= ensures that env vars can be supplied at the command line.
+
+# Global Lustre Root Directory - used for data movement tests
+GLOBAL_LUSTRE_ROOT=/lus/global
```

Comment:
Why do the variables below have a '?' but not this one?

Reply:
Ah, thanks for the catch. I'm rushing to get this to Brian.